examples/scenes/src/test_scenes.rs

```diff
@@ -366,6 +429,35 @@ fn tricky_strokes(scene: &mut Scene, _: &mut SceneParams) {
         );
         color_idx = (color_idx + 1) % colors.len();
     }
+
+    let flat_curves = [
+        flat_quad.as_slice(),
+        flat_conic_as_quads.as_slice(),
+        bigger_flat_conic_as_quads.as_slice(),
+    ];
+    for quads in flat_curves.iter() {
+        let mut path = BezPath::new();
+        path.push(MoveTo((1., 1.).into()));
+        for quad in quads.iter() {
+            path.push(QuadTo(quad[0].into(), quad[1].into()));
+        }
+        let x = (idx % NUM_COLS) as f64 * CELL_SIZE;
+        let y = (idx / NUM_COLS) as f64 * CELL_SIZE;
```

Comment:
These now go off screen at the default window size

Reply:
I can adjust the CELL_SIZE to adapt to the number of rows (which fixes this on my external monitor).

The layout in general is sensitive to DPI, e.g. at the default window dimensions, everything fits and looks tiny on my MacBook's retina display, but enlarged on my external monitor. I've observed this with all the other test scenes too.

Reply:
The test scene now initializes the `resolution` field of `SceneParams`, which fixes the issue for me.